<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReleaseTool</class>
 <widget class="QWidget" name="ReleaseTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_tool">
     <item row="8" column="0">
      <spacer name="horizontalSpacer_left">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="4">
      <spacer name="horizontalSpacer_rt">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="4">
      <widget class="QPushButton" name="pushButton_delete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <spacer name="horizontalSpacer_middle">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="3" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_length">
       <item>
        <widget class="QLabel" name="label_length">
         <property name="text">
          <string>Initial Length:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_length"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_length">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="7" column="2">
      <widget class="QPushButton" name="pushButton_copy">
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_spill_exp">
       <item>
        <widget class="QLabel" name="label_spill_exp">
         <property name="text">
          <string>Initial Spill Experience:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_spill_exp"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_spill_exp">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="3" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_species">
       <item>
        <widget class="QLabel" name="label_relspecies">
         <property name="text">
          <string>Species:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_species">
         <property name="text">
          <string>spec</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_species">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="4" column="1" colspan="3">
      <widget class="QSlider" name="horizontalSlider_start">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>365</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
       <property name="tickInterval">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_relstart">
       <property name="text">
        <string>Release Start:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_relsite">
         <property name="text">
          <string>Release Site:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_site">
         <property name="text">
          <string>site</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_segment">
       <item>
        <widget class="QLabel" name="label_relsegment">
         <property name="text">
          <string>River Segment:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_segment">
         <property name="text">
          <string>seg</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_segment">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="7" column="3">
      <spacer name="horizontalSpacer_mid_rt">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="4">
      <layout class="QHBoxLayout" name="horizontalLayout_start">
       <item>
        <widget class="QSpinBox" name="spinBox_start">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>365</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_reljulstart">
         <property name="text">
          <string>Julian Day</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="pushButton_schedule">
       <property name="text">
        <string>Show Schedule</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="5">
      <layout class="QHBoxLayout" name="horizontalLayout_existing">
       <item>
        <spacer name="horizontalSpacer_exist_lf">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_existing"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_exist_rt">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="7" column="1">
      <spacer name="horizontalSpacer_mid_lf">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="4">
      <widget class="QPushButton" name="pushButton_graph">
       <property name="text">
        <string>Show Graph</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="pushButton_new">
       <property name="text">
        <string>New</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_stock">
       <item>
        <widget class="QLabel" name="label_relstock">
         <property name="text">
          <string>Stock:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_stock">
         <property name="text">
          <string>stck</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_stock">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="0" colspan="5">
      <widget class="QLabel" name="label_existing">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Existing Releases&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBox_start</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_start</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_start</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_start</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
